.travis.yml

```diff
@@ -24,6 +24,7 @@ install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
       rvm install ruby-2.3.3;
       rvm --default use 2.3.3;
+      export DYLD_LIBRARY_PATH=/usr/local/opt/curl/lib:${DYLD_LIBRARY_PATH};
```

Comment:
Can you add a comment to explain why this line is added? It would make the future maintenance easier.

Reply:
Sure. I will add it after the tests run. I may need to set some back to pending.